thirdparty_builtin/CMakeLists.txt

```diff
@@ -154,10 +161,20 @@ if(ENABLE_TESTS)
                              TREAT_INCLUDES_AS_SYSTEM ON
                              )
 
+        list(APPEND _blt_thirdparty benchmark)
+
         # This sets up a target to run the benchmarks
         add_custom_target(run_benchmarks 
                           COMMAND ctest -C Benchmark -VV -R benchmark
                           WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
                           )
     endif()
+
+    # Set the folder property of the blt thirdparty libraries 
+    if(ENABLE_FOLDERS)
+        foreach(tpl ${_blt_thirdparty})
+            blt_set_target_folder(TO ${tpl} NAME blt/thirdparty)
```

Comment:
With the current incarnation --  I think folks may get confused and think that `TO` the directory not the target. 

I know it mirrors other args, but `TO` seems very strange in this context. 

The other methods are more like ``add things TO something``, where this is setting something about a target.

Reply:
I can change it to ``TARGET``